simphony/io/h5_cuds.py

```diff
@@ -152,6 +155,47 @@ def add_mesh(self, mesh):
         self._file.flush()
         return m
 
+    def add_lattice(self, lattice, record=None):
```

Comment:
We need a way of adding a lattice to a file with only a selected set of keywords, instead of all of them. This is currently done by using a custom record class that describes the Pytables columns. 

Only two solutions came to my mind how to solve this problem: 
1) Having an optional record parameter here, that's a reference to a custom IsDescription class that describes the columns and datatypes. [chosen]
2) Copying all the needed data from the original lattice to a new one, and adding that lattice to the H5CUDS-file. This gets rid of the record parameter in add_lattice, but it can multiply the memory requirements for the lattice (at the moment of creating the new one). Lattice-Boltzmann code is already run by filling the computer cluster node memory so every additional memory hungry operations should be avoided.

If you can come up with some other method of selecting the CUBA-keys, I'm interested in hearing that. :)